ext/fetch/22_body.js

```diff
@@ -92,6 +94,8 @@
         if (consumed) {
           this.streamOrStatic = new ReadableStream();
           this.streamOrStatic.getReader();
+          this.streamOrStatic[_disturbed] = true;
```

Comment:
Can you add a `readableStreamDisturb` function to `06_streams.js` instead so we can keep this symbol private to that module?

Reply:
done!